test/performance/benchmarks_test.go

```diff
@@ -148,7 +145,16 @@ func TestBenchmarkBurstZeroToN(t *testing.T) {
 func TestBenchmarkBurstNto2N(t *testing.T) {
 	for _, load := range loads {
 		t.Run(fmt.Sprintf("N%d", load), func(t *testing.T) {
-			runTest(t, app, float64(load), []float64{1, 2})
+			// Steady ramp up from 0 to N, then burst to 2N
+			zeroToNSteadyPacer := SteadyUpPacer{
+				Min:        vegeta.Rate{Freq: 1, Per: time.Second},
+				Max:        vegeta.Rate{Freq: load, Per: time.Second},
+				UpDuration: duration,
+			}
+			runTest(t, zeroToNSteadyPacer, false)
```

Comment:
Add a comment here that we're scaling up to the desired load and discard the metrics.

Reply:
Done.